.github/workflows/irso-functional.yml

```diff
@@ -0,0 +1,49 @@
+name: IrSO Functional Tests
+
+on:
+  pull_request:
```

Comment:
Also, this is now triggering on all branches, including release. The ci-e2e.sh scripts are not present in branches. So we need to either backport this, or limit it to main branch only.

/hold

Reply:
Pardon my ignorance, workflows are branchless, i.e. always taken from main? I definitely remember us backporting workflows.

Reply:
It depends which trigger is used, but in case of `pull_request` trigger, you're actually correct, it is triggered from the target branch and not the main like some others.